doc/ref_arch/kubernetes/chapters/appendix-a.rst

```diff
@@ -15,11 +15,17 @@ Kubernetes Namespace is a logical representation of namespace(boundary for resou
 This is different from the `linux namespaces <https://en.wikipedia.org/wiki/Linux_namespaces>`__ which are defined at
 the operating system kernel level.
 
-.. image:: ../figures/Model2-cluster-isolation.png
+.. figure:: ../figures/Model1-ns.png
+   :alt: "Network Service"
```

Comment:
I think qoutes are not required -- also line 19, 25 and 26

Reply:
Done in [cb8548a](https://github.com/cntt-n/CNTT/pull/3065/commits/cb8548a10cf600c00393c834448bf2ef384db51e)